src/main/kotlin/CountryLookup.kt

```diff
@@ -20,15 +20,18 @@ class CountryLookup {
                     return
                 }
 
-                val bytes = CountryLookup::class.java.classLoader.getResource(IP_TABLE_FILE)?.readBytes() ?: return
-                initializeWithBytes(bytes)
+                val classLoader = CountryLookup::class.java.classLoader
+                val resourceAsStream = classLoader.getResourceAsStream(IP_TABLE_FILE)
+
+                resourceAsStream?.use { inputStream ->
+                    val bytes = inputStream.readBytes()
+                    initializeWithBytes(bytes)
+                } ?: return
             }
         }
 
         @JvmStatic
         fun lookupIPString(ipAddressString: String): String? {
-            initialize()
```

Comment:
this might be a bad idea - the lock initialize should prevent this from doing multiple initializations.  I think we had to add this because in usage, we lazy init the ip lookup table, so having this here enforces that is complete before we do a lookup.  open to feedback